bench-vortex/src/bin/tpch.rs

```diff
@@ -35,7 +35,7 @@ struct Args {
     threads: Option<usize>,
     #[arg(long)]
     use_remote_data_dir: Option<String>,
-    #[arg(short, long, default_value = "5")]
+    #[arg(short, long, default_value_t = 10)]
```

Comment:
This is the change I actually wanted to make, everything was else was a result of my finally googling the difference between `default_value` and `default_value_t`